spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require 'simplecov'
 require 'simplecov-lcov'
+require_relative 'helper/board_pieces'
```

Comment:
don't expand unnecessarily the dependencies here. If this helper is being used only on a few spec files, import it on each file, not here

Reply:
done